### PR TITLE
[#111] Fix issue with requirements check on system configuration page

### PR DESCRIPTION
**Pull request by shadcn**
Issue: https://github.com/apigee/apigee-m10n-drupal/issues/111

### CODE REVIEW

**Comment by shadcn** [comment]
❌ Failure: https://circleci.com/gh/arshad/apigee-m10n-drupal/663
✅ Fixed: https://circleci.com/gh/arshad/apigee-m10n-drupal/668

File: apigee_m10n.install [hunk 1]
```diff
@@ -35,15 +35,20 @@ function apigee_m10n_requirements($phase) {
...
+      if (!$monetization->isMonetizationEnabled()) {
+        $requirements['apigee_monetization_connection_error'] = [
+          'title' => t('Apigee Monetization'),
+          'description' => t(Monetization::MONETIZATION_DISABLED_ERROR_MESSAGE),
+          'severity' => REQUIREMENT_ERROR,
+        ];
+      }
+    }
+    catch (Exception $exception) {
+
```
Comment (kscheirer) [inline apigee_m10n.install]: should probably do something with the exception, maybe log to watchdog?

Comment (Jaesin) [reply apigee_m10n.install]: I don't think we want to just hide all exceptions here. We should probably be catching `\Apigee\Edge\Exception\ApiException` here.

Comment (shadcn) [reply apigee_m10n.install]: Done and done.

**Review by Jaesin — APPROVED** [review]
This has been testing as is working.

**Pull request merged**